src/test/java/bogdanSandu/tests/Tema5/HerokuAppTests.java

```diff
@@ -0,0 +1,94 @@
+package bogdanSandu.tests.Tema5;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import java.util.List;
+
+public class HerokuAppTests {
+    private static ChromeDriver driver = null;
+    HerokuAppTests(String url, ChromeDriver driver){
+        HerokuAppTests.driver = driver;
+        driver.get(url);
+    }
+    public void returnPage(){
+        WebElement pageReturn = driver.findElement(By.linkText("Index"));
+        pageReturn.click();
+    }
+
+    //Tests for first link
+    public void firstLinkOpen(){
+        WebElement firstLink = driver.findElement(By.id("basicpagetest"));
+        firstLink.click();
+    }
+    public void getFirstLinkTitle(){
+        WebElement linkTitle = driver.findElement(By.xpath("//*[@id=\"basicpagetest\"]"));
+        System.out.println("First link title is: " + linkTitle.getText());
+    }
+    public void getFirstLinkText(){
+        WebElement linkText = driver.findElement(By.cssSelector("body > div > div.explanation"));
+        System.out.println("First text is: " + linkText.getText());
+    }
+    public void getFirstParagraph(){
+        WebElement paragraph = driver.findElement(By.id("para1"));
+        System.out.println("First paragraph is: " + paragraph.getText());
+    }
+
+    //Tests for second link
+    public void secondLinkOpen(){
+        WebElement secondLink = driver.findElement(By.id("elementattributestest"));
+        secondLink.click();
+    }
+    public void getSecondLinkTitle(){
+        WebElement secondPageTitle = driver.findElement(By.id("elementattributestest"));
+        System.out.println("Second link title is: " + secondPageTitle.getText());
+    }
+    public void getSecondtLinkText(){
```

Comment:
Misspelled Second word. Please double check the name of the method and rename it. Thanks!